packages/suite/.stylelintrc

```diff
@@ -1,5 +1,7 @@
 {
-    "processors": [],
+    "processors": [
+        "stylelint-processor-styled-components"
```

Comment:
do we need this? I had a feeling this is no longer necessary and stylelint should be able to lint styled components properly out of the box.

https://github.com/styled-components/stylelint-processor-styled-components/issues/278

Reply:
looks like we do (same as in @trezor/components) otherwise its failing on custom typed props:
```
src/components/suite/NavigationBar/components/MainNavigation/index.tsx
 35:5   ✖  Cannot parse selector              parseError 
 35:5   ✖  Cannot parse selector              parseError 
 35:5   ✖  Cannot parse selector              parseError 
 35:5   ✖  Cannot parse selector              parseError 
 35:5   ✖  Cannot parse selector              parseError 
 36:9   ✖  Expected indentation of 4 spaces   indentation
 37:9   ✖  Expected indentation of 4 spaces   indentation
 38:13  ✖  Expected indentation of 4 spaces   indentation
 39:9   ✖  Expected indentation of 4 spaces   indentation
 53:9   ✖  Expected indentation of 4 spaces   indentation
 54:9   ✖  Expected indentation of 4 spaces   indentation
 55:13  ✖  Expected indentation of 4 spaces   indentation
 56:9   ✖  Expected indentation of 4 spaces   indentation
```